prisma/schema.prisma

```diff
@@ -11,11 +11,11 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model USER {
-  id          String  @unique @default(uuid())
+model User {
+  id          String @unique @default(uuid())
   project_id  String
-  username    String?
-  email       String
+  username    String @unique
```

Comment:
Wouldn't `username` and `email` only be unique in the context of a project? (Meaning the same username could be used in different projects)

Reply:
I wasn't sure what the use case was. I assumed users would register once. If that's not the case, then I can change that. unique is required for Prisma's `findOne` api.

Reply:
Yea @gregfrasco can correct me if I'm wrong, but I believe the supported case is that there could be different instances of users with the same username/email to support different use cases for different projects

Reply:
I think there is a working prisma.schema in the main branch that has been reviewed for several times. Or do we have new changes to it? @gregfrasco

Reply:
Would it make more sense to separate out any project records with users? Else it's redundant to create a new row for each project id.

Reply:
That makes sense after my convo with Greg. Just changed this.